### DISCUSSION TITLE
Instant Replay

### DISCUSSION THREAD

**Discussion by MiguelElGallo** [ghostty-org/ghostty#3283] [Feature Requests, Ideas]
Instant Replay supported by iTerm for example is : Sometimes interactive programs will overwrite something of interest on the screen (for example, top(1) does this all the time). Normally, this would be lost forever. With Instant Replay, you can step back in time to see exactly what was on your screen at some point in the recent past.

**Comment by 00-kat** [comment]
Related (as it is for the same underlying issue): #3927.
